src/nvim/ui.h

```diff
@@ -66,4 +66,7 @@ struct ui_t {
 # include "ui.h.generated.h"
 # include "ui_events_call.h.generated.h"
 #endif
+
+
+EXTERN MultiQueue *resize_events;
```

Comment:
I wonder why `ch_before_blocking_events` declaration in `src/nvim/msgpack_rpc/channel.h` compiles without `extern`? Is it needed there?

OTOH, this compiles without EXTERN too.

Reply:
`EXTERN` is the standard pattern used by `global.h` and some other places for globals. It is only strictly required together with `INIT()`, but I guess it is good to be consistent ("any global declared in a .h file is marked with EXTERN").